Content.Server/CartridgeLoader/Cartridges/NanoTaskCartridgeSystem.cs

```diff
@@ -0,0 +1,177 @@
+using Content.Shared.CartridgeLoader;
+using Content.Shared.CartridgeLoader.Cartridges;
+using Content.Shared.Paper;
+using Robust.Shared.Utility;
+using Robust.Shared.Audio;
+using Robust.Server.Audio;
+using Content.Shared.Interaction;
+using Content.Shared.Hands.EntitySystems;
+using Robust.Shared.Log;
+using Content.Shared.PDA;
+
+namespace Content.Server.CartridgeLoader.Cartridges;
+
+public sealed class NanoTaskCartridgeSystem : EntitySystem
+{
+    [Dependency] private readonly CartridgeLoaderSystem _cartridgeLoader = default!;
+    [Dependency] private readonly AudioSystem _audio = default!;
```

Comment:
This code also needs to be in shared, all new code needs to be predicted.

Reply:
pretty much all of the cartridge system code lives in server, what i could move to shared is essentially just a skeleton